packages/app/features/home/screen.tsx

```diff
@@ -1,74 +1,32 @@
 import { Text, useSx, View, H1, P, Row, A } from 'dripsy'
+import { useState } from 'react'
 import { TextLink } from 'solito/link'
 import { MotiLink } from 'solito/moti'
+import SearchBar from './searchBar'
+import styles from './styles';
 
 export function HomeScreen() {
   const sx = useSx()
+  const [searchResults, setSearchResults] = useState([])
+
+  const handleSearch = async (searchText: String) => {
+    const response = await fetch(`https://api.dictionaryapi.dev/api/v2/entries/en/${searchText}`);
+    const data = await response.json();
+    setSearchResults(data)
+    console.log(searchResults)
```

Comment:
You are always logging the previous `searchResults` because the `setSearchResults` just above is **async**, that's very important to remember 😅 

The correct way to log the output would be to log `data`

Reply:
It's a log, so it doesn't matter much. But what we have here is definitely a bug

Reply:
done